system/datatypes/peer_reviewed_article.php

```diff
@@ -719,7 +719,12 @@ protected function statusChangeAction(HyphaRequest $request, $newStatus) {
 			return ['redirect', $this->constructFullPath($this->pagename)];
 		}
 
-		$success = $this->updateToNewStatus($newStatus);
+		list($checkStatus, $_checks) = $this->runChecks();
+		if ($checkStatus == self::CHECK_STATUS_FAILED) {
```

Comment:
Strict comparison is recommended.

Reply:
Still not fully in my system ;-)

Fixed.